api4/cloud_test.go

```diff
@@ -282,6 +284,56 @@ func Test_requestTrial(t *testing.T) {
 	})
 }
 
+func TestNotifyAdminToUpgrade(t *testing.T) {
+	t.Run("user can only notify admin once in cool off period", func(t *testing.T) {
```

Comment:
Any possibility to add a test that it will notify again after the cool off period? We can probably set a shorter time overriding the env var

Reply:
Added a test